site/tsdb-engine/css/demo.css

```
@@ -161,3 +161,562 @@
+  border-radius: 2px;
+}
+.legend-ts::before { background: color-mix(in srgb, #6366f1 40%, transparent); border: 1px solid #6366f1; }
+.legend-vals::before { background: color-mix(in srgb, #10b981 40%, transparent); border: 1px solid #10b981; }
+.legend-frozen::before { background: color-mix(in srgb, #3b82f6 40%, transparent); border: 1px solid #3b82f6; }
+.legend-hot::before { background: color-mix(in srgb, #f59e0b 40%, transparent); border: 1px solid #f59e0b; }
+.legend-ts-spine::before { background: color-mix(in srgb, #6366f1 40%, transparent); border: 1px solid #6366f1; }
+.legend-val-col::before { background: color-mix(in srgb, #10b981 40%, transparent); border: 1px solid #10b981; }
+
+/* ── Query Lab ──────────────────────────────────────────────────────── */
+
+.query-panel { }
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)